image_decompress.py

```diff
@@ -4,14 +4,9 @@
 import numpy as np
 
 centers = np.load('codebook_tiger.npy')
-
 c_image = io.imread('compressed_tiger.png')
+image = centers[c_image]

```

Comment:
Wow, this line of code really do the same job after re-run the code,
and shape is 
```python
print("centers", centers.shape)
print("c_image", c_image.shape)
print("image", image.shape)

output:
centers (128, 3)
c_image (720, 1280)
image (720, 1280, 3)
```
So, can you explain this numpy trick or any related material.